src/Objs/Font/FontEditingConfig.ts

```diff
@@ -0,0 +1,63 @@
+import { Binary, provideEditingConfig } from 'scrivito'
+import { Font } from './FontObjClass'
+
+provideEditingConfig(Font, {
+  title: 'Font',
+  attributes: {
+    family: { title: 'font-family' },
+    variations: {
+      title: 'Variable font?',
+      description:
+        'Does this file contain multiple font variations? See https://fonts.google.com/knowledge/glossary/variable_fonts for details.',
+    },
+    weight: {
+      title: 'font-weight',
+      description:
+        'Either a single value (e.g. "400") or a range (e.g. "100 1000").',
+    },
+  },
+  properties: ['family', 'weight', 'variations'],
+  validations: [
+    [
+      'blob',
+      (blob: Binary | null) => {
+        if (blob?.raw().contentType() === 'font/woff2') return
+
+        return {
+          message: 'Only WOFF2 font files are supported.',
```

Comment:
I'm hesitant to add ttf support, because `ttf` files are much larger then `woff2` files (often more then double the size). Since font file size also play a critical role during first page visits I opted to only support the most condense format.

Reply:
We are not speaking about real life use. This font feature is for demo cases / prototyping. At least with the current set of abilities.

Reply:
> ttf files are much larger then woff2 files (often more then double the size).
Keep in mind that our CDN has compression enabled. Actually I expect fonts to have a similar network size regardless of the format. WOFF is precompressed, which will improve things a little since they can pick a higher/slower compression. I just tested it with Fira Sans Regular and the TTF (441 K uncompressed file) download size is 195 K (according to cURL), while the WOFF2 in the repo is 147 K.

Reply:
Thanks for the pointer that our CDN is compressing TTF quite nicely. With this (and the fact that we are not longer declaring a specific font type) I relaxed the validation to enforce, that it is of mime type `font/`.